app/src/main/java/com/sunasterisk/food_01/data/source/remote/api/ParseDataWithJson.kt

```diff
@@ -53,8 +54,9 @@ class ParseDataWithJson {
         try {
             jsonObject?.let {
                 when (keyEntity) {
-                    RecipeEntry.RECIPE -> PasreJson().recipeParseJson(jsonObject)
-                    else -> null
+                    RecipeEntry.RECIPE -> return PasreJson().recipeParseJson(jsonObject)
+                    CategoryEntry.OBJECT -> return PasreJson().categoryParseJson(jsonObject)
+                    else -> return null
                 }
             }
         } catch (e: JSONException) {
```

Comment:
Đặt return trước try 
return try { } catch 
https://kotlinlang.org/docs/reference/exceptions.html#try-is-an-expression